test/e2e/ginkgo/multiple_cluster_deployment_test.go

```diff
@@ -562,6 +563,46 @@ var _ = ginkgo.Describe("end to end testing", func() {
 			gomega.Expect(*cs.Results[0].Rows[0].Columns[0].Number).To(gomega.Equal("2"))
 		})
 
+		ginkgo.It("can list all the WECs where the reported number of availableReplicas equals the desired number of replicas", func(ctx context.Context) {
+
+			availableNginxCEL := ksapi.Expression("obj.spec.replicas == returned.status.availableReplicas")
+			util.CreateStatusCollector(ctx, ksWds, listNginxWecsStatusCollectorName,
+				ksapi.StatusCollectorSpec{
+					Filter: &availableNginxCEL,
+					Select: []ksapi.NamedExpression{
+						{
+							Name: "wecName",
+							Def:  "inventory.name",
+						},
+					},
+					Limit: 10,
+				})
+
+			testAndStatusCollection[0].StatusCollectors = []string{listNginxWecsStatusCollectorName}
+			util.CreateBindingPolicy(ctx, ksWds, bpName, clusterSelector, testAndStatusCollection)
+
+			cs := util.GetCombinedStatus(ctx, ksWds, wds, ns, workloadName, bpName)
+
+			// Validate CombinedStatus results
+			gomega.Expect(len(cs.Results)).To(gomega.Equal(1))
+			gomega.Expect(cs.Results[0].Name).To(gomega.Equal(listNginxWecsStatusCollectorName))
+
+			gomega.ExpectWithOffset(1, len(cs.Results[0].ColumnNames)).To(gomega.Equal(1))
+			gomega.Expect(cs.Results[0].ColumnNames[0]).To(gomega.Equal("wecName"))
+
+			gomega.ExpectWithOffset(1, len(cs.Results[0].Rows)).To(gomega.Equal(2))
```

Comment:
The offset should be zero here, as in the other Expect calls in this func.
Can be pursued in follow-on.